superset/views/core.py

```diff
@@ -894,7 +900,7 @@ def schemas(self, db_id: int, force_refresh: str = "false") -> FlaskResponse:
     @has_access_api
     @expose("/tables/<int:db_id>/<schema>/<substr>/")
     @expose("/tables/<int:db_id>/<schema>/<substr>/<force_refresh>/")
-    def tables(
+    def tables(  # pylint: disable=too-many-locals,no-self-use
```

Comment:
Whoa, I'm surprised it's flagging `no-self-use` on these. Perhaps they should be disabled for the whole file if they cause false positives on Flask endpoints like this..

Reply:
I'd love to disable it for whole files, but I'd like us to agree on a "No non-flask-endpoint functions/methods in flask endpoint files" rule first :)